### ISSUE TITLE
Add option to add labels to table

### ISSUE THREAD

**Issue by djvanderlaan** [djvanderlaan/tinymarkdown#2]
Perhaps if omitted and the code-block has an id and the table has a caption, use the id of the code-block.

When the table has no caption the function should give a warning and ignore the request to add a label.

**Comment by djvanderlaan** [comment]
This is something that is not supported yet in pandoc markdown. It is not possible to add an id (and classes etc) to tables. Will keer this open for now in case pandoc fixes this.

**Comment by emeralit** [comment]
With pandoc-crossref filter adding to the caption {tbl:label} is enough.

**Comment by djvanderlaan** [comment]
Thanks. I know. However, as far as I know it doesn't come standard with the default pandoc installation. That's why I didn't yet include it. Perhaps I could add the option and add in the documentation that this feature only works with the plugin installed. On the other hand there are workarounds:

````
```{.R fun=output_table caption="Sample iris {#id}"}
iris[1:5, ]
```
````
or

````
: Caption {#id}

```{.R fun=output_table}
iris[1:5, ]
```
````
Let me know if you think it is better to add this to the package (I don't think I am against it :-) ).

**Comment by emeralit** [comment]
I agree, the real issue is as pandoc-crossref manages labels. Please, add the examples to the documentation,
Thanks a lot for your job.
